helmfile/charts/prometheus-alerts/templates/alerts/opensearch.yaml

```diff
@@ -34,14 +34,6 @@ spec:
       annotations:
         description: The heap usage is over 90% for 15m
         summary: OpenSearch node {{`{{ $labels.node}}`}} heap usage is high
-    - alert: OpenSearchDiskWarning
```

Comment:
Do we have another alert that checks the pvc? Or why can we remove this alert?

Reply:
because I didn't find the metrics for it: es_fs_total_free_bytes and es_fs_total_total_bytes
pvc size should be covered by [this alerts](https://github.com/elastisys/compliantkubernetes-apps/blob/update-alerts-rules/helmfile/charts/prometheus-alerts/templates/alerts/kubernetes-storage.yaml)
and node disk size by [this alerts](https://github.com/elastisys/compliantkubernetes-apps/blob/update-alerts-rules/helmfile/charts/prometheus-alerts/templates/alerts/node-exporter.yaml)